app/controllers/provider_interface/organisation_permissions_setup_controller.rb

```diff
@@ -9,7 +9,48 @@ def index
         current_provider_user,
       )
       @sorted_and_grouped_provider_names = permission_setup_presenter.grouped_provider_names
-      @sorted_permission_ids = permission_setup_presenter.sorted_provider_permission_ids
+      @sorted_relationship_ids = permission_setup_presenter.sorted_provider_permission_ids
+      OrganisationPermissionsSetupWizard.new(organisation_permissions_wizard_store, relationship_ids: @sorted_relationship_ids).save_state!
```

Comment:
Since we are relying on this action to initialize the list of relationships, maybe we need to add a redirect for the other actions (edit, update, check) that send users back to the start page when the wizard store doesn't contain a list of relationship ids.

Reply:
spoke in person: we should redirect and also have an error message (possibly similar to the make offer flow)

Reply:
Asked design. Confirmed we can just redirect to the start of the flow if the cache expires and do not require a flash message